### PR TITLE
wip(tenant-setup): init setup tasks

### PR DESCRIPTION
**Pull request by ipcrm**

### CODE REVIEW

File: plans/util/integrate_lw_dockerhub.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      prompt: false
+  description: |
+    This plan integrates a Lacework tenant with a Docker Hub account
+  steps:
+    - name: integrate-lacework-with-dockerhub
+      extension: Terraform
+      tags: lacework
+      source:
+        location: "git+https://github.com/lacework-dev/detc-resources.git"
+        subdir: "util/tenant_setup/terraform/dockerhub"
```
Comment (ipcrm) [inline plans/util/integrate_lw_dockerhub.yaml:24]: Is this pushed?

**Review by ipcrm — COMMENTED** [review]
I think some of this wasn't pushed up?

File: plans/util/integrate_lw_dockerhub.yaml (hunk 1 above)
Comment (gabeobrien) [reply plans/util/integrate_lw_dockerhub.yaml:24]: Is it now...!

File: plans/util/ansible/update-cspm-policies.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+    - name: Enable all Lacework Policies
```
Comment (ipcrm) [inline plans/util/ansible/update-cspm-policies.yml:2]: I think we need to add these too:

```
lacework compliance gcp disable CIS_1_2
lacework compliance azure disable CIS_1_3_1
```

File: util/tenant_setup/terraform/dockerhub/main.tf [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+variable "dockerhub_password" {}
+
+provider "lacework" {
+  subaccount = var.lacework_subaccount
+  account    = var.lacework_account
+  api_key    = var.lacework_apikey
+  api_secret = var.lacework_apisecret
+}
+
+resource "lacework_integration_docker_hub" "example" {
```
Comment (ipcrm) [inline util/tenant_setup/terraform/dockerhub/main.tf:24]: nit: change this from example

File: plans/util/integrate_lw_dockerhub.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+Dockerhub:
```
Comment (ipcrm) [inline plans/util/integrate_lw_dockerhub.yaml:2]: This file should exist under the util/ path; not as a subdirectory of plans.

File: plans/util/ansible/update-cspm-policies.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (ipcrm) [inline plans/util/ansible/update-cspm-policies.yml:1]: This file should exist under /util not in the plans dir

File: plans/util/ansible/enable-policies.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (ipcrm) [inline plans/util/ansible/enable-policies.yml:1]: This file should be in /util not in the plans directory

**Review by ipcrm — COMMENTED** [review]
@gabeobrien looking good!

I made a couple minor comments.

Final thought, I think _some_ of the items in tenant_setup.yaml can be run before app deployment, some have to be run after.

We should probably break this up accordingly.

File: plans/util/integrate_lw_dockerhub.yaml (hunk 4 above)
Comment (gabeobrien) [reply plans/util/integrate_lw_dockerhub.yaml:2]: But this file is a plan. it is the plan to setup the docker hub integration.

**Comment by gabeobrien** [comment]
@ipcrm I split out the steps into two plans as you requested.  The **tenet-setup** plan only needs a LW account and can be run when ever.  The **post_deployment_setup** plan needs to be run after the deployment has completed.

File: plans/util/integrate_lw_dockerhub.yaml (hunk 4 above)
Comment (ipcrm) [reply plans/util/integrate_lw_dockerhub.yaml:2]: 🤦 My bad @gabeobrien

**Review by gabeobrien — APPROVED** [review]
+1

**Pull request merged**